src/Shared/NewSwapModal.razor

```diff
@@ -0,0 +1,503 @@
+                                    </Tooltip>
+                                </FieldLabel>
+                                <div class="d-flex">
+                                    <NumericPicker TValue="int"
+                                                @bind-Value="_maxMinerFeesPercent"
+                                                CurrencySymbolPlacement="CurrencySymbolPlacement.Suffix"
+                                                CurrencySymbol="%"
+                                                Min="0"
+                                                Max="100">
+                                    </NumericPicker>
+                                </div>
+                                <FieldHelp>
+                                    @{
+                                        var amount = _maxMinerFeesPercent * _amountBtc / 100;
+                                    }
+                                    @Math.Round(amount, 8) BTC (~@Math.Round(PriceConversionService.BtcToUsdConversion(amount, _btcPrice), 2) USD)
+                                </FieldHelp>
+                            </Field>
+                            <Field>
+                                <FieldLabel Class="d-flex">
+                                    Max Routing Fees
+                                    <Tooltip Class="ml-2" Text="Maximum routing fees in satoshis you are willing to pay for the lightning routing transaction">
+                                        <Icon Name="IconName.Info"></Icon>
+                                    </Tooltip>
+                                </FieldLabel>
+                                <div class="d-flex">
+                                    <NumericPicker TValue="int"
+                                                @bind-Value="_maxRoutingFeesPercent"
+                                                CurrencySymbolPlacement="CurrencySymbolPlacement.Suffix"
+                                                CurrencySymbol="%"
+                                                Min="0"
+                                                Max="100">
+                                    </NumericPicker>
+                                </div>
+                                <FieldHelp>
+                                    @{
+                                        var amount = _maxRoutingFeesPercent * _amountBtc / 100;
+                                    }
+                                    @Math.Round(amount, 8) BTC (~@Math.Round(PriceConversionService.BtcToUsdConversion(amount, _btcPrice), 2) USD)
+                                </FieldHelp>
+                            </Field>
+                            <Field>
+                                <FieldLabel Class="d-flex">
+                                    Sweep confirmation target
+                                    <Tooltip Class="ml-2" Text="Target number of confirmations for swaps, this uses bitcoin core broken estimator, procced with caution">
```

Comment:
IIRC, it was fixed and I think, pending conf, that it uses LND's? and LNDs one we have a custom conf calculator, worth checking in loop repo

Reply:
changed